src/_includes/assets/css/screen.css

```
@@ -2409,6 +2409,11 @@ a.banner:hover span {
   direction: rtl;
 }
 
+.rtl-layout .post-full-content pre,
+.rtl-layout .post-full-content code {
```

```suggestion
.rtl-layout .post-full-content pre {
```
I don't think styling the `code` is necessary.

---

Awesome, sounds good.

You're right -- it doesn't seem like the `p code` elements are affected either way.